### PR TITLE
Feature/listserv front end

### PR DESCRIPTION
**Pull request by Vittorio2015**
As we discussed earlier, I created new classes and customized the buttons.

### CODE REVIEW

File: listserv/index.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        } );
+    </script>
+
+  </head>
+  <body class="lti-tool" role="application">
+    <header>
+      <h1>Course Mailing List</h1>
+    </header>
+    <main class="float-left col-60">
+        <p style="padding-bottom: 2em">
```
Comment (roderickmorales) [inline listserv/index.html]: Need to remove all inline styling

File: listserv/index.html [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        <h3>
+            CSCI E-12 (21144) mailing list address
+        </h3>
+        <div style="padding-left: 1em">
+            <p>
+                <spam class="listserv-caption-status">
+                    The mailing list address contains everyone in the course independent of roles, enrollment status and credit status.
+                </spam>
+            </p>
+            <main class="float-left col-50">
```
Comment (roderickmorales) [inline listserv/index.html]: You can only have one main tag per page.

File: listserv/index.html [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+            CSCI E-12 (21144) sections mailing list addresses
+        </h3>
+        <div style="padding-left: 1em;">
+            <p>
+                <spam class="listserv-caption-status">
+                    These sections are created from the <a href="" target="_blank">Manage Section</a> tool of this course and any changes applied there will be reflected within these lists.
+                </small>
+            </p>
+            <main class="listserv-main-content">
+                <main class="float-left col-50">
```
Comment (roderickmorales) [inline listserv/index.html]: the <main> tag is to be used only once in a page. You can read a little more about it here: http://www.w3schools.com/tags/tag_main.asp

File: listserv/index.html [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+                    <p>
+                        <spam class="listserv-caption">
+                            <strong>
+                             Second section (15 members)
+                             </strong>
+                            <br>
+                            <spam class="listserv-caption-disabled">
+                                <strong>
```
Comment (roderickmorales) [inline listserv/index.html]: You are misusing the strong tag here. Don't use html tags to style the page use css, this type of markup gives the wrong accessibility to the page.

File: listserv/index.html [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+                                    Mailing list address:
+                                </strong>
+                                disabled
+                        </spam>
+                    </p>
+                </main>
+                <main class="float-left col-50">
+                    <p>
```
Comment (roderickmorales) [inline listserv/index.html]: You don't need to add the button inside a paragraph tag. I think you are using the paragraph tag to give vertical spacing, you should be using CSS for styles.

File: listserv/index.html [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+                        </spam>
+                    </p>
+                </main>
+            </main>
+        </div>
+
+        </main>
+
+    </body>
+</html>
```
Comment (roderickmorales) [inline listserv/index.html]: Add an extra line at the bottom for source control.

File: listserv/index.html (hunk 2 above)
Comment (Vittorio2015) [inline listserv/index.html]: As we just discussed, I'll leave only one <main> tag and use <div> for the other instances of <main> and <p>.

File: listserv/index.html (hunk 3 above)
Comment (Vittorio2015) [inline listserv/index.html]: As we just discussed, I'll leave only one main tag and use div for the other instances of main and p.

File: listserv/index.html (hunk 4 above)
Comment (Vittorio2015) [inline listserv/index.html]: Got it. I'll create new classes to define these types of information.

File: listserv/index.html (hunk 5 above)
Comment (Vittorio2015) [inline listserv/index.html]: As we just discussed, I'll use div when needed, instead of p.

File: listserv/index.html (hunk 6 above)
Comment (Vittorio2015) [inline listserv/index.html]: Done

File: listserv/index.html [hunk 7]
```diff
@@ -0,0 +1,137 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Course Mailing List</title>
+
+    <link rel="stylesheet" href="../../django-icommons-ui/icommons_ui/static/lti-css/listserv.css">
+    <link rel="stylesheet" type="text/css" href="../../django-icommons-ui/icommons_ui/static/font-awesome-4.2.0/css/font-awesome.min.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
+    <!--[if lt IE 9]>
```
Comment (elliottyates) [inline listserv/index.html]: What processes this? Is it a browser-level thing?

File: listserv/index.html [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
+    <![endif]-->
+    <script type="text/javascript" language="javascript" src="../../django-icommons-ui/icommons_ui/static/datatables-1.10.7/media/js/jquery.js"></script>
+
+    <script type="text/javascript" language="javascript" class="init">
```
Comment (elliottyates) [inline listserv/index.html]: This looks like it shouldn't be on the listserv index page, did it creep in from a page copy?

**Comment by elliottyates** [comment]
Should `listserv/index.html` be the only file in this feature request? The others seem to be related to bulk site creation.

File: listserv/index.html (hunk 7 above)
Comment (rascalking) [inline listserv/index.html]: It's an IE-specific thing, I think...I'm used to seeing that syntax around IE-targeted polyfill stuff.

https://msdn.microsoft.com/en-us/library/ms537512(v=vs.85).aspx#dlhidden

Comment (elliottyates) [inline listserv/index.html]: Thanks Dave, I had not seen this before.

**Comment by roderickmorales** [comment]
@elliottyates just catching up with emails... this is Vittorio first HTML integration so we'll be tighten things up. I think the main thing to get out of this exercise is to get him comfortable with branching, committing, PR's, etc.
Thank you for looking at it and we'll take care of all this things by tomorrow.

**Comment by roderickmorales** [comment]
:+1:

**Pull request merged**